File: UtilitiesVB/Interfaces/IToDoObjects.vb

Public Interface IToDoObjects
    ReadOnly Property DictPPL As IPeopleDict
    ReadOnly Property DictRemap As Dictionary(Of String, String)
    ReadOnly Property IDList As IListOfIDs
    ReadOnly Property Parent As IApplicationGlobals
    ReadOnly Property ProjInfo As IProjectInfo
    ReadOnly Property FnameProjectInfo As String
    ReadOnly Property FnameDictPeople As String
    ReadOnly Property FnameDictRemap As String
    ReadOnly Property FnameIDList As String

End Interface

File: UtilitiesCS/Interfaces/IToDoObjects.cs
using System.Collections.Generic;
using UtilitiesCS;

namespace UtilitiesCS
{

    public interface IToDoObjects
    {
        Dictionary<string, string> DictPPL { get; }
        string DictPPL_Filename { get; }
        void DictPPL_Save();
        Dictionary<string, string> DictRemap { get; }
        ISerializableList<string> CategoryFilters { get; }
        IListOfIDsLegacy IDList { get; }
        IApplicationGlobals Parent { get; }
        IProjectInfo ProjInfo { get; }
        string ProjInfo_Filename { get; }
        string FnameDictRemap { get; }
        string FnameIDList { get; }

    }
}